recipe/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "av" %}
+{% set version = "8.0.2" %}
+{% set sha256 = "a3bba6bf68766b8a1a057f28869c7078cf0a1ec3207c7788c2ce8fe6f6bd8267" %}
+{% set build_number = 1 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build_number }}
```

Comment:
This should be 1 instead of `{{ build_number }}`

Reply:
Oh, I missed that `build_number was defined up above. It would match the rest of our recipes if we don't use a variable here.

Reply:
sure, updated it.